ui/test/nuxeo-date-picker.test.js

```diff
@@ -32,6 +36,20 @@ function testValue(element, value, isUTC) {
   // expect(element._pickerValue).to.be.equal(local);
 }
 
+function testValueWithLocale(element, value, locale, isUTC) {
+  const currentLocale = moment.locale();
+  moment.locale(locale);
+  element.value = value;
+  expect(element.value).to.be.equal(value);
+  const inputValue = getInputDisplay(element).value;
+  let val = isUTC ? moment.utc(value) : moment(value).local();
+  // use the i18n functions we set in nuxeo-date-picker, since the displayed date is not the value
+  val = getInput(element).i18n.formatDate(getInput(element).i18n.parseDate(val));
+  expect(inputValue).to.be.equal(val);
+  // reset the locale for next tests
+  moment.locale(currentLocale);
```

Comment:
@Gabez0r imagining the test fails, should we put the locale reset in a hook method, lets say the `afterEach`? Maybe that means I should add a new suite, right? wdyt about it?